ffi/ecma402/src/list.rs

```diff
@@ -0,0 +1,92 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use core::fmt::{self, Display, Write};
+use ecma402_traits::listformat::{
+    options::{Style, Type},
+    Format, Options,
+};
+use ecma402_traits::Locale;
+use writeable::Writeable;
+
+pub struct ListFormat(icu::list::ListFormatter);
+
+impl Format for ListFormat {
```

Comment:
Nit: I would find this much more clear if you fully qualified the trait
```suggestion
impl ecma402_traits::listformat::Format for ListFormat {
```
This is how it is done in pluralrules.rs